pca.go

```diff
@@ -0,0 +1,81 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+)
+
+// PrincipalComponents returns the k principal component direction vectors of
+// the matrix a, projected score vectors and variance values by singular value
+// decomposition of the input. The input a is a matrix where each observation
```

Comment:
Make it `... where each row is an observation ... `? Would be consistent with `... each column represents a variable.`.